app/pods/publications/publication/documents/filter-query-params.js

```diff
@@ -0,0 +1,81 @@
+import { set } from '@ember/object';
```

Comment:
Misschien hoort dit meer thuis onder /utils?

Reply:
Deze component ziet er redelijk generiek uit, er staat geen publication only code in.
Dus ja, kan op zich wel onder /utils, als het duidelijk blijft dat deze component een filter is op documenten
utils/documents/filter-query-params ?

Reply:
Deze constructie oogt me op het eerste zicht wat ongewoon. Ik verwachtte eerder iets te zien als [hier](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/2464ab2a408108a49d34b00d960b6e06fca27746/app/pods/search/agenda-items/route.js#L77-L110), zie ook bijhorende controller. Bekijk ook zeker eens het `array` type [queryParameter](https://api.emberjs.com/ember/3.17.0/classes/Controller/properties/queryParams?anchor=queryParams). Eens dit meer bewerkt is in bovenstaande richting, kan deze file in z'n geheel verdwijnen en deel worden van de `model` hook lijkt me.